src/main/java/org/alien4cloud/plugin/kubernetes/modifier/KubernetesFinalTopologyModifier.java

```diff
@@ -417,7 +463,7 @@ private void manageVolume(FlowExecutionContext ctx, Csar csar, Topology topology
         volumeEntry.put(PropertyUtil.getScalarValue(volume_type), volumeSpecObject);
         feedPropertyValue(deploymentResourceNodeProperties, "resource_def.spec.template.spec.volumes", volumeEntry, true);
 
-        NodeType volumeNodeType = ToscaContext.get(NodeType.class, volumeNode.getType());
+        //NodeType volumeNodeType = ToscaContext.get(NodeType.class, volumeNode.getType());
```

Comment:
This has been moved above, delete this line instead of commenting it out, otherwise it may be confusing